validators/builtin-validators.mdx

```diff
@@ -287,6 +288,55 @@ Example:
 
 You can find this example in [Javascript](https://github.com/Lemoncode/fonk/tree/master/examples/docs/api/js/email) and [Typescript](https://github.com/Lemoncode/fonk/tree/master/examples/docs/api/ts/email).
 
+## Array
+
+This validator allows you to add support to validation to array form fields, it work as an inception,
+you get the fonk engine embedded into that array, _e.g. you got a shopping cart list and you want
+to validate that each Quantity field on each row is informed and is a valid number_.
+
+```typescript
+export interface ArrayArgs {
+  field: FieldValidationSchema;
+}
+
+interface ArrayValidationResult {
```

Comment:
Remove from 302 to 312 lines, it's an internal interface

Reply:
done